cmdstanpy/install_cmdstan.py

```diff
@@ -10,17 +10,16 @@
    -v, --version <release> : version, defaults to latest release version
    -d, --dir <path> : install directory, defaults to '$HOME/.cmdstan
    --overwrite: flag, when specified re-installs existing version
-   --verbose: flag, when specified prints output from CmdStan build process
```

Comment:
Should readd this to docstring

Reply:
added to docstring, and further edits to better explain options and defaults.